Description: Alter max_matches in sphinx.conf
 Move max_matches from 1000 to 10000
 .
 securityonion-elsa-extras (20131117-1ubuntu0securityonion12) precise; urgency=low
 .
   * Correct BRO_HTTP ordering to preserve previous logs
Author: Scott Allen Runnels <srunnels@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- securityonion-elsa-extras-20131117.orig/bin/securityonion-elsa-config.sh
+++ securityonion-elsa-extras-20131117/bin/securityonion-elsa-config.sh
@@ -124,6 +124,8 @@ function config_lognode() {
 
     # Ensure the correct pid file
 	sed -i "s|pid_file = /var/run/searchd.pid|pid_file = /var/run/sphinxsearch/searchd.pid|" /etc/sphinxsearch/sphinx.conf
+    # Correct the max_matches entry
+    sed -i 's|max_matches = 1000|max_matches = 10000|' /etc/sphinxsearch/sphinx.conf
 
 	# Set up directory locks
 	mkdir -p "$BASE_DIR/elsa/node/tmp/locks"
